src/core/entities/MRTextAreaEntity.js

```diff
@@ -12,56 +12,65 @@ import { MRTextInputEntity } from 'mrjs/core/entities/MRTextInputEntity';
 export class MRTextAreaEntity extends MRTextInputEntity {
     /**
      * @class
+     * @description Constructor for the MRTextInputEntity entity component.
      */
     constructor() {
         super();
-        // Define additional properties for handling multiline text and scrolling
         this.lineHeight = 1.2; // Default line height, can be adjusted as needed
-        this.scrollOffset = 0; // The vertical scroll position
-        this.maxVisibleLines = 10; // Maximum number of lines visible without scrolling
         this.object3D.name = 'textArea';
     }
 
     /**
-     *
+     * @function
+     * @description Called by connected to make sure the hiddenInput dom element is created as expected.
      */
     createHiddenInputElement() {
+        // setup
         const inputElement = document.createElement('textarea');
+
+        // style
         inputElement.style.position = 'absolute';
-        inputElement.style.left = '-9999px'; // Position off-screen
+        inputElement.style.left = '-9999px'; // Position off-screen so dont have weird white spot at top
         inputElement.style.height = '1px';
         inputElement.style.width = '1px';
         inputElement.style.overflow = 'hidden';
-        document.body.appendChild(inputElement); // Ensure it's part of the DOM for event capturing
+
+        // Ensure it's part of the DOM for event capturing
+        document.body.appendChild(inputElement);
```

Comment:
I think we talked about this but why aren't we putting this in the shadow root? might scale a little better. especially for embedded experiences.

Reply:
oh no main reason - i can swap it to that

Reply:
Yeah it shouldn't be a huge issue and then you wouldn't need to do the 9999999/-99999999 absolute positions. you can just set it to `display:none` in CSS.

Reply:
that is so much cleaner haha

Reply:
Yeah use this as an example https://github.com/Volumetrics-io/mrjs/pull/578#discussion_r1563401938

It's one of those stack overflow tricks I picked up the first time I was trying with MRTextField.